pkgs/servers/openafs/default.nix

```diff
@@ -48,7 +55,7 @@ stdenv.mkDerivation rec {
     maintainers = [ maintainers.z77z ];
     broken =
       (builtins.compareVersions kernel.version  "3.18" == -1) ||
```

Comment:
If you care to, these raw calls to `compareVersions` have more readable wrappers in `versionAtLeast` & `versionOlder` and friends (see lib/strings.nix).